ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +575,44 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    use_cudnn_on_gpu=False,
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D"
+
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 3 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+
+    if padding == 'EXPLICIT':
+        input = Explicit_Padding(input, paddings=(1, 1, 1, 1))
+        padding = 'VALID'
+
+    return tf_frontend.nn.conv2d(
+        input,
+        filter,
+        strides,
+        padding,
+        data_format=data_format,
+        dilations=dilations,
+        name=name,
+    )
+
```

Comment:
This is not the correct approach. You should use `ivy.conv2d` that can handle explicit padding. Basically, if `padding == 'EXPLICIT'` then set `padding=explicit_paddings` and pass it to `ivy.conv2d`

Reply:
should we return the output

Reply:
You shouldn't be using `tf_frontend.nn.conv2` at all, but return the `ivy.conv2d` output. Also `padding` would be set equal to the value of `explicit_paddings`, not a string "explicit_paddings". 
Please read the documentation of the involved functions carefully and try to understand their operations. The way you are using them right now doesn't make much sense.

Reply:
I was taking inspiration from conv3D

Reply:
i am not able to find any mention of explicit_paddings in doc

Reply:
[conv3d](https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conv3D) doesn't handle explicit padding like [conv2d](https://www.tensorflow.org/api_docs/python/tf/raw_ops/Conv2D). 
`padding=='explicit'` means that the user defines how many pixels will be padded before and after each dimensions. The number of those pixels is given as the argument `explicit_paddings'. That's what happends in `tf.raw_ops.Conv2d`.
Our [conv2d](https://lets-unify.ai/ivy/functional/ivy/layers/conv2d.html?highlight=conv2d) has an argument `padding` that can be either "same", or "valid" (as in  `tf.raw_ops.Conv2d`), or a sequence of numbers (this corresponds to `tf.raw_ops.Conv2d`'s `explicit_paddings`.

Reply:
yeah i understood